modules/tinymce/src/core/main/ts/init/InitIframe.ts

```diff
@@ -52,6 +53,8 @@ const createIframeElement = (id: string, title: TranslatedString, height: number
     title
   });
 
+  tabindex.each((t) => Attribute.set(iframe, 'tabindex', t));
```

Comment:
```suggestion
  Attribute.setOptions(iframe, {
    tabindex
  });
```
I realised we could do this later and I think it's a bit cleaner?

Reply:
Note: was later reverted as didn't quite work the same